### PR TITLE
Feature/about us

### PR DESCRIPTION
**Pull request by stevekevin1005**
model/test(FAQ完成）
controler部分因還未切出service因此還未做

### CODE REVIEW

**Comment by smlsunxie** [comment]
Can one of the admins verify this patch?

File: api/services/FAQService.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+module.exports = {
+	test : async() => {
+		console.log("test");
+	}
```
Comment (clonn) [inline api/services/FAQService.js]: 等一個 service ....

**Comment by clonn** [comment]
@stevekevin1005  merge function.

**Comment by clonn** [comment]
left spec , please pull request again.

**Pull request merged**